src/methods/clipping/difference.jl

```diff
@@ -28,7 +28,7 @@ GI.coordinates.(diff_poly)
 """
 function difference(
     geom_a, geom_b, ::Type{T} = Float64; target::Type{Target} = Nothing,
-) where {T <: AbstractFloat, Target <: GI.AbstractTrait}
+) where {T <: AbstractFloat, Target <: Union{Nothing, GI.AbstractTrait}}
```

Comment:
Why `Nothing` here ?

Reply:
Right now, if the target isn't specified then I have it set to `Nothing` as I am not sure what else it should be. I have a vision for that to change and have the input types determine the minimum viable union. So for example, if a target isn't provided and the inputs are polygons, then the target should be a union of points, lines, and polygons. But I haven't implemented that so the default is `Nothing` right now.